pkg/gornir/gornir.go

```diff
@@ -16,6 +19,89 @@ type Gornir struct {
 	Logger    Logger     // Logger for the object
 }
 
+// NewGornir is a Gornir constructor.
+func NewGornir() *Gornir {
```

Comment:
I'd personally just skip this, it doesn't provide much value

Reply:
Yeah, it doesn't provide any functionality right now. However is a placeholder to insert any defaults in the future.